src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala

```diff
@@ -17,21 +19,26 @@ import scala.concurrent.{ExecutionContext, Future}
 object TezosPlatformDiscoveryOperations {
 
   type AttributesCache = Map[String, (Long, List[Attribute])]
+  type AttributeValuesCache = Map[String, (Long, RadixTree[String, String])]
   type EntitiesCache = (Long, List[Entity])
 
   def apply(metadataOperations: MetadataOperations,
     attributesCache: MVar[IO, AttributesCache],
     entitiesCache: MVar[IO, EntitiesCache],
+    attributeValuesCache: MVar[IO, AttributeValuesCache],
```

Comment:
Yes, I plan to do it in a improvement PR which will change a bit behaviour of the cache, but also I think it will be a better place to perform refactor